static/src/javascripts/projects/common/modules/atoms/youtube.spec.js

```diff
@@ -48,4 +74,250 @@ describe('youtube', () => {
             expect(document.body.innerHTML).toBe(div);
         }
     });
+
+    describe('muting iframes to trgiger autoplay', () => {
+        it('mutes an iframe by amending the src property if no other URL parameters', () => {
+            const docSrc = 'http://www.testme.com/q';
```

Comment:
Not sure about using a domain out of our control for this... 🤔 
If we must, can we use `example.com` ?